src/style.css

```diff
@@ -0,0 +1,38 @@
+body {
+  background-color: black;
+  margin: 0;
+}
+
+.blue-box {
+  width: 350px;
+  background-color: #308dc0;
+  margin-bottom: 40px;
+  margin-left: 20px;
+  border-width: 10px;
+  border: 1px solid #e75551;
+  box-sizing: border-box;
+  padding: 10px;
+}
+
+.yellow-box {
+  background-color: #ffa500;
+  padding: 10px;
+  border: 1px solid #e75551;
+  box-sizing: border-box;
+}
+
+.white-box {
+  background-color: #ddd;
+  padding: 10px;
+  border: 1px solid  #e75551;
+  box-sizing: border-box;
```

Comment:
The theme of the site may be changed and binding class names to colors is bad practice. You can think of these blocks as content, wrapper, and inner wrapper. Example: outer, inner, wrapper, content, etc.

Reply:
fixed